trite-osprey/src/List.js

```diff
@@ -5,7 +5,8 @@ class List extends React.Component{
     super(props)
     this.state = {
```

Comment:
Maybe, when you click a beer, you could store the data for that beer in the state and then render the beerinfo component by passing it that data.